lib/libyui_client/app.rb

```diff
@@ -28,7 +38,7 @@ def button(filter)
     # @param filter [Hash] filter to find a widget
     # @return [Widgets::Checkbox] new instance of Checkbox
     # @example
-    #   app.checkbox(id: 'id', label: 'label', type: 'YCheckBox')
+    #   app.checkbox(id: 'id', label: 'label', class: 'YCheckBox')
```

Comment:
we could mention in [checkbox.rb](https://github.com/jknphy/libyui_client/blob/master/lib/libyui_client/widgets/checkbox.rb#L17) that we also cover YCheckBoxFrame.

Reply:
I don't like the way we document things currently, so I was not sure if example is a good way to show that certain type of widget is supported.

Reply:
I've updated rubydoc of checkbox class and added example making it more clear that both widgets act the same. I guess that should be enough.